src/site/stages/build/plugins/add-nonce-to-scripts.js

```diff
@@ -1,4 +1,4 @@
-const jsdom = require('jsdom');
```

Comment:
This file had to change quite a bit because jsdom does things a little differently from cheerio.